### PR TITLE
Merge codes in linear-concise

### PR DESCRIPTION
**Pull request by mli**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_linear-networks/linear-regression-scratch.md [hunk 1]
```diff
@@ -87,12 +83,15 @@ def synthetic_data(w, b, num_examples):  #@save
     """Generate y = Xw + b + noise."""
     X = tf.zeros(shape=(num_examples, w.shape[0]))
     X += tf.random.normal(shape=X.shape)
-    y = tf.matmul(X, w) + b
+    y = tf.matmul(X, tf.reshape(w, (-1,1))) + b
```
Comment (astonzhang) [inline chapter_linear-networks/linear-regression-scratch.md:86]: `(-1,1)` -> `(-1, 1)`

Comment (astonzhang) [reply chapter_linear-networks/linear-regression-scratch.md:86]: can we use d2l.matmul?

Comment (mli) [reply chapter_linear-networks/linear-regression-scratch.md:86]: we can't due to `tf.random.normal`

**Comment by mli** [comment]
Job d2l-en/PR-1182/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1182/

**Pull request merged**